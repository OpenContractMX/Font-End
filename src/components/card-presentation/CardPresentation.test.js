import React from "react";
import "@testing-library/jest-dom/extend-expect";
import { render } from "@testing-library/react";
import { CardPresentation } from "./";

it("[render content `component / CarPresentation`]", () => {
  const profileImg = "img",
    name = "name",
    jobTitle = "text",
    twitterProfile = "link",
    linkedinProfile = "link";
  const { asFragment } = render(
    <CardPresentation
      profileImg={profileImg}
      name={name}
      jobTitle={jobTitle}
      twitterProfile={twitterProfile}
      linkedinProfile={linkedinProfile}
    />
  );
  expect(asFragment()).toMatchSnapshot();
});
