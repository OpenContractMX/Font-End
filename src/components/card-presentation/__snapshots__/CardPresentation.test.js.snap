// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[render content \`component / CarPresentation\`] 1`] = `
<DocumentFragment>
  <article
    class="card-presentation"
  >
    <figure
      class="card-presentation__image-card"
    >
      <img
        alt=""
        src="img"
      />
       
    </figure>
    <p
      class="card-presentation__title-name"
    >
      name
    </p>
    <p
      class="card-presentation__title-job-title"
    >
      text
    </p>
    <div
      class="card-presentation__container-icons"
    >
      <span
        class="card-presentation__container-icons-linkedin-icon"
      >
        <a
          href="link"
          rel="noreferrer"
          target="_blank"
        >
          <i
            class="fab fa-linkedin-in"
          />
        </a>
      </span>
      <span
        class="card-presentation__container-icons-twitter-icon"
      >
        <a
          href="link"
          rel="noreferrer"
          target="_blank"
        >
          <i
            class="fab fa-twitter"
          />
        </a>
      </span>
    </div>
  </article>
</DocumentFragment>
`;
